### PR TITLE
ft(add database of signin) - add signin api [finishes 169538006]

### PR DESCRIPTION
**Pull request by ishoborabyose**
#### What does this PR do?
This pull request is for create signin database
#### Description of Task to be completed?
- insert  codes in user controller
#### How should this be manually tested?
- Clone the repo
- Cd into the project folder folder
- npm run dev

#### Any background context you want to provide?
Not Applicable
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/169538006

### CODE REVIEW

File: server/controllers/user.controller.js [hunk 1]
```diff
@@ -60,40 +61,37 @@ export const signup = async (req, res) => {
...
-//         lastname,
-//         email
-//       }
-//     });
-//   }
- };
+    const authtoken = jwt.sign({ id: uuid.v1(), }, process.env.MY_SECRET, {
+      expiresIn: "2d"
+    });
+    return res.status(201).json({
```
Comment (frankhn) [inline server/controllers/user.controller.js]: Should we be returning 201 on a successful login? uuh!! check it up: return 200 instead

File: server/controllers/user.controller.js [hunk 2]
```diff
@@ -60,40 +61,37 @@ export const signup = async (req, res) => {
...
-//     return res.status(400).send({
-//       status: 400,
-//       error: "User with provided email or password do not exist!"
-//     });
-
-//   const token = jwt.sign({ email: newUser.email }, process.env.MY_SECRET, {
-//     expiresIn: "2d"
-//   });
+export const signin = async (req, res) => {
+    let { email, password } = req.body;
```
Comment (frankhn) [inline server/controllers/user.controller.js]: Are you re-assigning these elements? Please, review the scoping here. should be const instead

**Review by frankhn — CHANGES_REQUESTED** [review]
Kindly, Review my comments and refactor the changes.

File: server/controllers/user.controller.js (hunk 2 above)
Comment (ishoborabyose) [reply server/controllers/user.controller.js]: I collected it

File: server/controllers/user.controller.js (hunk 1 above)
Comment (ishoborabyose) [reply server/controllers/user.controller.js]: I collected it

**Review by frankhn — APPROVED** [review]
LGTM.

**Review by frankhn — COMMENTED** [review]
This PR has lots of changes, consider refactoring and following GIT flow.

**Pull request merged**